techniques/failures/F23.html

```diff
@@ -1,40 +1,60 @@
-<!DOCTYPE html><html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml"><head><title>Failure of  1.4.2 due to playing a sound longer than 3 seconds where
-                    there is no mechanism to turn it off</title><link rel="stylesheet" type="text/css" href="../../css/sources.css" class="remove"></link></head><body><h1>Failure of  1.4.2 due to playing a sound longer than 3 seconds where
-                    there is no mechanism to turn it off</h1><section class="meta"><p class="id">ID: F23</p><p class="technology">Technology: failures</p><p class="type">Type: Failure</p></section><section id="applicability"><h2>When to Use</h2>
+<!DOCTYPE html>
+<html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+   <title>Failure of  1.4.2 due to playing a sound longer than 3 seconds where there is no mechanism to turn it off</title>
```

Comment:
"down" suggests to me lower volume is okay. If you want it more precise "muted" is probably the word to use.

Reply:
I agree with Wilco that turn down isn't specific enough as the SC requires stopping not just lowering the volume.  Turn down doesn't say by how much.

Reply:
In that case, I suggest we leave as is, since there is little practical difference between muting and turning off. The SC does allow lowering the volume as a mechanism, provided lowering can go all the way to zero, but for concision, I'm going to just resolve this without changing.